test/jdk/javax/swing/JPopupMenu/7156657/bug7156657.java

```diff
@@ -38,8 +37,6 @@
 import javax.swing.JPopupMenu;
 import javax.swing.SwingUtilities;
 
-import sun.awt.SunToolkit;
-
```

Comment:
The "@modules java.desktop/sun.awt" should be removed as well?

Reply:
Yes, sorry, forgot about it. Fixed.